IdentityCore/src/cache/key/MSIDDefaultAccountCacheQuery.m

```diff
@@ -50,4 +50,9 @@ - (BOOL)exactMatch
     return self.homeAccountId && self.environment && self.realm;
 }
 
+- (BOOL)isShared
```

Comment:
nit: remove this, already implemented in the parent class

Reply:
Done.